src/test/java/com/wikia/webdriver/common/testnglisteners/BrowserAndTestEventListener.java

```diff
@@ -50,115 +46,130 @@ public void beforeNavigateTo(String url, WebDriver driver) {
   @Override
   public void afterNavigateTo(String url, WebDriver driver) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "afterNavigateTo" is 24 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)